ec2net-functions

```diff
@@ -107,25 +114,24 @@ get_meta() {
   done
 
   # IMDS can take up to 30s to provide the information of a new ENI
-  attempts=60
+  attempts=${max_attempts}
   imds_exitcode=1
   while [ "${imds_exitcode}" -gt 0 ]; do
```

Comment:
Should this be `-ne` instead of `-gt` to avoid some weird edge case with a negative exit code?

Reply:
Bash will always return exit codes in the 0-255 range.

Reply:
TIL. I'm mixing up internal functional programming with bash.